src/main/java/be/seeseemelk/mockbukkit/inventory/meta/BookMetaMock.java

```diff
@@ -250,14 +252,10 @@ public void setPages(String... pages)
 	public void setPages(@NotNull List<String> pages)
 	{
 		this.pages.clear();
-		Iterator<String> var2 = pages.iterator();
-
-		while (var2.hasNext())
-		{
-			String page = var2.next();
-			this.addPage(page);
-		}
 
+        for (String page : pages) {
```

Comment:
This causes a ConcurrentModificationException, no? We should have tests for all methods here, whether in this PR or another.

Reply:
IntelliJ gave me that hint and I thought it made sense.
It shouldn't modify anything?

```
        for (String page : pages) {
            this.addPage(page);
        }
```

And in `addPage` you just overwrite the string if it's too long, and add it into the map. So nothing being modified in there as I see? (but my knowledge of Java is lacking, so I am probably wrong)

Reply:
Ahh that's the list passed into the method, not the one in the class; that naming isn't confusing at all.

It should be fine then, but should still have tests.

Reply:
Can you suggest which additional tests would be needed?

Reply:
Ok @Thorinwasher : I implemented the requested tests.